code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -768,11 +768,10 @@ var/global/list/animal_count = list() //Stores types, and amount of animals of t
 				adjustBruteLoss(volume * 0.5)
 
 /mob/living/simple_animal/proc/delayedRegen()
-	set waitfor = 0
 	isRegenerating = 1
```

Comment:
It does in life()

It's what PJB told me to do, but I'm gonna move it to just after the revive and see what happens